### DISCUSSION TITLE
regex pattern based file read and multiple files read when regex pattern is matched

### DISCUSSION THREAD

**Discussion by Siddanagouda22** [nightscape/spark-excel#888] [General]
I am using this library. I have few questions I want to know regular expression pattern based file reading is supported by this plugin or not. 
and also 
if file pattern is matched with multiple files can we read all the files matched. 

Please let me know this plugin support above use case 

Thanks in advance

**Comment by nightscape** [comment]
In v2 (`.format("excel")`) we should support the same options there as the built-in CSV library as we build upon the same base classes.
